src/enum_net.cpp

```diff
@@ -1052,7 +1052,7 @@ namespace libtorrent {
 		nl_msg->nlmsg_len = NLMSG_LENGTH(sizeof(rtmsg));
 		nl_msg->nlmsg_type = RTM_GETROUTE;
 		nl_msg->nlmsg_flags = NLM_F_DUMP | NLM_F_REQUEST;
-		nl_msg->nlmsg_seq = seq++;
+		nl_msg->nlmsg_seq = seq;
```

Comment:
I went to double check my info on `nlmsg_pid` and went down a bit of a rabbit hole. I'll add another commit with some more changes and my findings.